### ISSUE TITLE
Character sounds not playing

### ISSUE THREAD

**Issue by CrocodileCroco** [ikemen-engine/Ikemen-GO#1427]
### Describe the bug

Some characters seems to be muted, not playing any sound and i have no error showing in the console.

### To Reproduce

1. Download a character from whatever mugen website and add it to your Ikemen GO game
2. Select that character and start the round

### Expected behavior

The characters should be playing their sounds from the .snd file

### Engine Version (or source code date)

v0.99.0-rc.3

### Operating system

Linux

### Extra context or search terms

I'm on Linux Mint latest version

**Comment by Lazin3ss** [comment]
Upload the character we should use for testing, otherwise we won't be able to fix this bug. In any case, it probably was the PlaySnd bug @Eiton fixed, try nightly version.

**Comment by CrocodileCroco** [comment]
It's this one which does not play any sound : https://www.dropbox.com/s/x0k2qnp04nh7hb0/SMG4%201.1.0.rar?dl=0

**Comment by Eiton** [comment]
The sounds of the character are played normally, at least on Windows build. Did you check the audio setting? Maybe you muted Ikemen Go.

**Comment by CrocodileCroco** [comment]
> The sounds of the character are played normally, at least on Windows build. Did you check the audio setting? Maybe you muted Ikemen Go.

Yes i did check the audio settings, Ikemen Go is not muted and when using the Windows build with wine, it still do the same thing (not playing character sounds) :/

**Comment by Eiton** [comment]
Dose ikemen go play other sounds? e.g. bgm, select menu sounds.
Does it happen on 0.98.2?
